packages/pilot/src/components/Operations/operationsTableColumns.js

```diff
@@ -77,11 +78,30 @@ const renderTargetOrSource = (type, net, targetId, sourceId, labels) => {
   return null
 }
 
+const renderBradescoTransferStatus = (type, status, outgoing, labels) => {
```

Comment:
```suggestion
const renderAccountTransferStatus = (type, status, outgoing, labels) => {
```
Acho que faz mais sentido tratar como credito em conta de forma generica e não atrelar a banco apesar de hoje usarmos bradesco o que acha?,

Reply:
Faz sentido! Alterei para `renderCreditTransferStatus`